### PR TITLE
Drop yast2-sound module.

### PR DESCRIPTION
**Pull request by jreidinger**
It's usefulness is reduced so much now, that it is basically useless except corner cases and other existing tools already do similar or better job.

## Problem

The last useful thing (mapping of hardware and kernel module ) in yast2-sound module stop working. So it is basically completely replaced by tools like alsamixer or pulseaudio mixer.

## Solution

drop module, but keep SLE-15 branches for maintenance. If it should be revived in future, SLE 15 branch can be used as base.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-[![Workflow Status](https://github.com/yast/yast-sound/workflows/CI/badge.svg?branch=master)](
-https://github.com/yast/yast-sound/actions?query=branch%3Amaster)
-[![Jenkins Status](https://ci.opensuse.org/buildStatus/icon?job=yast-yast-sound-master)](
-https://ci.opensuse.org/view/Yast/job/yast-yast-sound-master/)
+**This module is obsoleted and is not developed anymore.**
+
+GIT
+---
+
+Use the [*SLE-15-SP5*](../SLE-15-SP5) branch for SLE-15 maintenance.
```
Comment (teclator) [inline README.md:9]: Is this link to the **SLE-15-SP5** branch still correct? I mean, also checked other Readme for already dropped modules and them fail in the same way.

Comment (jreidinger) [reply README.md:9]: yeah, I worry github somehow changed branch structures and we just copy it over. So if someone is interested enough for fixing it, it should be fixed everywhere.

Comment (teclator) [reply README.md:9]: An what about using the explicit link instead? https://github.com/yast/yast-sound/tree/SLE-15-SP5

**Review by teclator — APPROVED** [review]
LGTM

**Pull request merged**